Fix Aurora's Website URL

asyf16 opened the pull request:
I changed my website's URL a few months ago and forgot to change it here! 

Info:
Full name: Aurora Shi
Year: 2028
Website URL: https://aurorashi.com/

simcard0000 reviewed (changes requested):
@asyf16 Thanks for the update! Not sure if I just missed it LOL, but do you have a linkback to the webring somewhere on your site?

asyf16 commented:
> @asyf16 Thanks for the update! Not sure if I just missed it LOL, but do you have a linkback to the webring somewhere on your site?

just added!! forgot to add it when i updated my website last week :))

simcard0000 reviewed (approved):
Perfect, thanks!
